<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>591</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mothboardTab">
     <attribute name="title">
      <string>Motherboard</string>
     </attribute>
     <widget class="QLabel" name="lblPrice">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMinPrice">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMaxPrice">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10000</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMinPrice">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMaxPrice">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblProducer">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Producer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbxProducer">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Asrock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASUS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BioStar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gigabyte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSI</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblFormFactor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Form-factor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbxFormFactor">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>E-ATX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Micro ATX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini ITX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Standard ATX</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblProcTypeMB">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Processor Type</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnIntelMB">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Intel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnAmdMB">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AMD</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Processor</string>
     </attribute>
     <widget class="QLabel" name="lblPrice_proc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnIntel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Intel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMaxPrice_proc">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMinPrice_proc">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMinPrice_proc">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMaxPrice_proc">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10000</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblProducer_proc">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Producer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnAmd">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AMD</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbxProducer_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cores</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RAM">
     <attribute name="title">
      <string>RAM</string>
     </attribute>
     <widget class="QComboBox" name="cmbxProducer_ram">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>AMD Radeon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dell</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hynix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kingston</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Samsung</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Transcend</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblMinPrice_ram">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMinPrice_ram">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMaxPrice_ram">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10000</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblProducer_ram">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Producer</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPrice_ram">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMaxPrice_ram">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMemory">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Memory</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblFrequency">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbxMemory">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>512 Mb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 Gb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 Gb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 Gb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 Gb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 Gb</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="ledtMinPrice_ram_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledtMaxPrice_ram_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>3300</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMaxPrice_ram_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMinPrice_ram_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>376</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Final Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
